src/forms/FilterForm/FilterForm.js

```diff
@@ -68,7 +81,12 @@ const FilterFormComponent = props => {
           >
             <div className={classNames(paddingClasses || css.contentWrapper)}>{children}</div>
 
-            {liveEdit ? (
+            {useDebounce && liveEdit ? (
```

Comment:
`KeywordFilter` is using our generic filter component so I added an option to use debounce in the generic `FilterForm`.  I don't know if it's ok or should I create new specific form etc. for `KeywordFilter`.

Reply:
I moved this debounce to KeywordFilter component